### PR TITLE
feat(docs): add table sorting

### PR DESCRIPTION
**Pull request by zethussuen**

### CODE REVIEW

File: docs/reference/filter.md [hunk 1]
```diff
@@ -140,11 +138,11 @@ Used with the `SHOW_ONLY_BRANDS` and `SHOW_ONLY_MODELS` variables.
 | `corsair` | `750 platinum`, `600 platinum` |
 | `evga` | `ftw3`, `ftw3 ultra`, `xc3`, `xc3 black`, `xc3 ultra` |
 | `gainward` | `phantom gs`, `phoenix`, `phoenix gs`, `phoenix gs oc` |
-| `gigabyte` | `aorus master`, `aorus xtreme`, `aorus xtreme waterforce`, `eagle`, `eagle oc`, `gaming`, `gaming oc`, `turbo`, `vision`, `vision oc` |
-| `inno3d` | `gaming x3`, `ichill x3`, `ichill x4`, `ichill frostbite`, `twin x2 oc` |
+| `gigabyte` | `aorus master`, `aorus xtreme`, `eagle`, `eagle oc`, `gaming`, `gaming oc`, `turbo`, `vision`, `vision oc` |
+| `inno3d` | `gaming x3`, `ichill x3`, `ichill x4`, `twin x2 oc` |
 | `kfa2` | `sg`, `sg oc` |
 | `microsoft` | `xbox series x`, `xbox series s` |
-| `msi` | `gaming x trio`, `suprim x`, `ventus 2x oc`, `ventus 3x`, `ventus 3x oc` |
```
Comment (jef) [inline docs/reference/filter.md:147]: Maybe have been a mistake, but we need these still.

**Review by jef — COMMENTED** [review]
Alright, I'm gonna pull this down and see what this looks like!

**Comment by jef** [comment]
Awesome! I totally missed this.

https://squidfunk.github.io/mkdocs-material/reference/data-tables/#sortable-tables

This is pretty sweet!

File: docs/reference/filter.md [hunk 2]
```diff
@@ -27,90 +27,88 @@
...
+| Micro Center | US | `microcenter`|
+| Mindfactory | DE | `mindfactory` |
+| Newegg | US | `newegg`|
+| Newegg | CA | `newegg-ca`|
+| Notebooksbilliger | DE |`notebooksbilliger`|
+| Novatech | UK | `novatech`|
+| Nvidia | US | `nvidia`|
+| Nvidia | API | `nvidia-api`|
+| Office Depot | US | `officedepot`|
+| Otto | ? | `otto`|
```
Comment (jef) [inline docs/reference/filter.md:94]:
```suggestion
| Otto | DE | `otto`|
```

File: docs/reference/filter.md [hunk 3]
```diff
@@ -27,90 +27,88 @@
...
+| Currys | UK | `currys`|
+| Cyberport | DE | `cyberport` |
+| eBuyer | UK | `ebuyer`|
+| El Corte Inglés | US | `elcorteingles`|
+| ePrice | IT | `eprice`|
+| Euronics | IT | `euronics`|
+| Euronics | DE | `euronics-de`|
+| EVGA | US | `evga`|
+| EVGA | EU | `evga-eu`|
+| Expert | ? | `expert`|
```
Comment (jef) [inline docs/reference/filter.md:76]:
```suggestion
| Expert | DE | `expert`|
```

File: docs/reference/filter.md [hunk 4]
```diff
@@ -27,90 +27,88 @@
...
+| Medimax | US | `medimax`|
+| MemoryExpress | CA | `memoryexpress`|
+| Micro Center | US | `microcenter`|
+| Mindfactory | DE | `mindfactory` |
+| Newegg | US | `newegg`|
+| Newegg | CA | `newegg-ca`|
+| Notebooksbilliger | DE |`notebooksbilliger`|
+| Novatech | UK | `novatech`|
+| Nvidia | US | `nvidia`|
+| Nvidia | API | `nvidia-api`|
```
Comment (jef) [inline docs/reference/filter.md:92]:
```suggestion
| Nvidia API | | `nvidia-api`|
```

File: docs/reference/filter.md (hunk 1 above)
Comment (zethussuen) [reply docs/reference/filter.md:147]: Oops yeah probably didn't rebase correctly.

**Review by jef — CHANGES_REQUESTED** [review]
Also, to fix the build, can you add

```json
,
  "include": ["src/**/*"]
```

to `tsconfig.json`

**Comment by jef** [comment]
You can ignore my comment on the `tsconfig.json`. I pushed that up to `main`. Thanks!

**Comment by zethussuen** [comment]
@jef any guidance on how you want to handle the js file that's included from the repo example? fails build-and-lint PR checks as it is not proper ts.

**Comment by zethussuen** [comment]
@jef Assuming no new stores are added, this should be good for review as of 4:40pm Eastern. There were a few that were ingested from rebases: `walmart-ca`, `saturn`, and `toysrus`. I also added an xo-linter rule that ignores the `docs/javascripts` folder. Up to you how you want to handle though!

**Comment by R3PTIL3** [comment]
Otto and Expert = DE 👍🏼

**Comment by jef** [comment]
> I also added an xo-linter rule that ignores the `docs/javascripts` folder.

Good idea!

**Review by jef — APPROVED** [review]
Looks great! Thank you very much :)

**Pull request merged**